core-services/prow/02_config/openstack-k8s-operators/ci-framework/_pluginconfig.yaml

```diff
@@ -2,7 +2,8 @@ approve:
 - commandHelpLink: ""
   repos:
   - openstack-k8s-operators/ci-framework
-  require_self_approval: false
+  require_self_approval: true
+  lgtm_acts_as_approve: false
```

Comment:
it seems that OpenShift CI will block us from merging this configuration. the `make prow-config` target deletes this line:
`-  lgtm_acts_as_approve: false`

Reply:
Based on the discussion with Stephen from DPTP team, `lgtm_acts_as_approve: false` is the default so no need to mention it and our patches were getting merged because approvers were using the github option to approve PRs, `ignore_review_state: true` will help us with that.

more about this parameter is here: https://github.com/kubernetes/test-infra/blob/master/prow/plugins/config.go#L316-L319

~~~
	// IgnoreReviewState causes the approve plugin to ignore the GitHub review state. Otherwise:
	// * an APPROVE github review is equivalent to leaving an "/approve" message.
	// * A REQUEST_CHANGES github review is equivalent to leaving an /approve cancel" message.
~~~

Reply:
:+1: